share/spice/nobel_prize/nobel_prize.js

```diff
@@ -0,0 +1,141 @@
+            for (var laureate of item.laureates) {
+                names.push(laureate.firstname + " " + (laureate.surname || ""));
+            }
+            
+            return names;
+        }
+
+        // Render the response
+        Spice.add({
+            id: "nobelprize",
+
+            // Customize these properties
+            name: "Nobel Prize",
+            data: api_result.prizes,
+            meta: {
+                sourceName: "nobelprize.org",
+                sourceUrl: 'http://nobelprize.org/',
+                itemType: 'Nobel Prizes',
+                searchTerm: getSearchTerm()
+            },
+            normalize: function(item) {
+                var title, description, imageURL,
+                    laureates = item.laureates;
+                
+                imageURL = getImageURL(item);
+                title = laureates[0].firstname + " " + (laureates[0].surname || "");
+                description = laureates[0].motivation;
+
+                return {
+                    title: title,
+                    icon: imageURL,
+                    altSubtitle: getSubtitle(laureates),
+                    description: description,
+                    year: item.year,
+                    category: item.category.toUpperCase(),
+                    detail_title_names: getDetailTitleNames(item)
+                };
+            },
+    
+            onItemShown: function(item) {
+                var laureate_info = {}, laureates = [];
+                laureate_info['overallMotivation'] = item.overallMotivation;
+
+                for(var laureate of item.laureates) {
+                    $.getJSON('http://api.nobelprize.org/v1/laureate.json?&id=' + laureate.id, function(laureate) {
+                        laureate = laureate.laureates[0];
+
+                        var prize;
+                        for(var p of laureate.prizes) {
+                            if (item.year == p.year)
+                                prize = p;
+                        }
+
+                        laureates.push({
+                            bornOn: laureate.born == "0000-00-00" ? 0 : (new Date(laureate.born).toDateString()),
+                            bornCity: laureate.bornCity,
+                            bornCountry: laureate.bornCountry,
+                            diedOn: laureate.died == "0000-00-00" ? 0 : (new Date(laureate.died).toDateString()),
+                            name: laureate.firstname + " " + (laureate.surname || ""),
+                            share: prize.share,
+                            motivation: prize.motivation
+                        });
+                    });
+                 }
+
+                laureate_info['laureates'] = laureates;
+                item.set({laureate_info: laureate_info})
+            },
+
+            templates: {
+                group: 'text',
+                options: {
+                    rating: false,
+                    moreAt: false,
```

Comment:
`moreAt` should be true. We should send the user to the nobel website for more info